src/client/Microsoft.Identity.Client/Platforms/netcore/NetCorePlatformProxy.cs

```diff
@@ -172,25 +173,33 @@ public override Task StartDefaultOsBrowserAsync(string url)
                     throw new MsalClientException(MsalError.LinuxXdgOpen, MsalErrorMessage.LinuxOpenAsSudoNotSupported);
                 }
 
+                List<string> openTools = new List<string>();
```

Comment:
Duplicate, unused, can be removed.

Reply:
Missed to remove. Thanks for catching that